hrms/patches/v15_0/check_version_compatibility_with_frappe.py

```diff
@@ -0,0 +1,20 @@
+import click
+
+import frappe
+
+
+def execute():
+	frappe_v = frappe.get_attr("frappe" + ".__version__")
+	hrms_v = frappe.get_attr("hrms" + ".__version__")
+
+	WIKI_URL = "https://github.com/frappe/hrms/wiki/Changes-to-branching-and-versioning"
+
+	if "14" in frappe_v and "15" in hrms_v:
```

Comment:
can cause false positive if minor version matches in future: `15.14` or `14.15`

Reply:
https://github.com/frappe/hrms/pull/294 fixed 💀. Thanks for pointing out!